bigtable/client/internal/table_admin.h

```diff
@@ -116,6 +119,20 @@ class TableAdmin {
                         bigtable::ConsistencyToken const& consistency_token,
                         grpc::Status& status);
 
+  std::future<bool> WaitForConsistencyCheck(
+      bigtable::TableId const& table_id,
+      bigtable::ConsistencyToken const& consistency_token,
+      std::unique_ptr<bigtable::PollingPolicy> polling_policy) {
+    return std::async(std::launch::async,
+                      &TableAdmin::WaitForConsistencyCheckImpl, this, table_id,
+                      consistency_token, std::move(polling_policy));
+  }
+
+  bool WaitForConsistencyCheckImpl(
```

Comment:
This should be a private member function I think.

Reply:
This function is called from the client/table_admin.cc and checking if exception is thrown,so this function is not made private

Reply:
You can make `bigtable::TableAdmin` a friend class and keep it private, better to have a small, controlled violation of access control, than a fully open case.  But see my other comment about having these functions in the `noex` case.